ext_finger_controller:
  controller:
    module: joint_position_controller
    package: dynamixel_controllers
    type: JointPositionController
  joint_name: ext_finger_joint
  joint_speed: 0.8
  motor:
    id: 18
    init: 740
    max: 875
    min: 735

int_finger_controller:
  controller:
    module: joint_position_controller
    package: dynamixel_controllers
    type: JointPositionController
  joint_name: int_finger_joint
  joint_speed: 0.8
  motor:
    id: 17
    init: 455
    max: 320
    min: 460